pkgs/by-name/ta/task-master-ai/package.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildNpmPackage,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+buildNpmPackage rec {
```

Comment:
We recommend using `finalAttrs` for builders that supports it, due to the many quirks that using `rec` can have. See:
- #315337